From 586041df94ad310e54159911b2a83cd7c5183939 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 30 Apr 2015 13:47:17 +0800
Subject: [PATCH 0108/5242] MLK-10783-6 doc: dt-binding: usbmisc_imx: add
 compatible string "fsl,imx6ul-usbmisc"

commit  d144b87d595df04fc34ef3486839f0edeb079759 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add compatible string "fsl,imx6ul-usbmisc"

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit 2bccb52eba125db06a7a7da5342d41c9f664d2d7)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/usb/usbmisc-imx.txt        |    1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/usb/usbmisc-imx.txt b/Documentation/devicetree/bindings/usb/usbmisc-imx.txt
index a85a631..0ff9017 100644
--- a/Documentation/devicetree/bindings/usb/usbmisc-imx.txt
+++ b/Documentation/devicetree/bindings/usb/usbmisc-imx.txt
@@ -7,6 +7,7 @@ Required properties:
 	"fsl,vf610-usbmisc" for Vybrid vf610
 	"fsl,imx6sx-usbmisc" for imx6sx
 	"fsl,imx7d-usbmisc" for imx7d
+	"fsl,imx6ul-usbmisc" for imx6ul
 - reg: Should contain registers location and length
 
 Examples:
-- 
1.7.9.5

